### ISSUE TITLE
Feature: Show 'inherited users' in Admin folder

### ISSUE THREAD

**Issue by willpopski** [meshifyiot/carbon-docs#89]
Currently, the only way to see users on a folder is to see users that are exactly in the folder.

This request is for a way to see all users who would inherit access to the folder because they are in a folder that contains (a folder that explains ... ) the given folder.

Use Case: Notification debugging and operational management. It will allow a support user to look directly at a folder that has a node in alert and verify that a given user should have access.

For UI, this could be a section like the current users section.

Assuming this has an API component, we'd want to integrate this into a custom HSB template.

**Comment by NateDawg90** [comment]
Is the current dashboard 3 situation of showing numbers of users in child folders enough? Or do we need more?

**Comment by willpopski** [comment]
No, this is about inherited access to a folder. So if I'm a root, my name doesn't appear on any other folders in Dashboard 3, yet I have access to that folder.

This feature request is to show a list of all users that have access to a folder.
